zcash_client_backend/src/data_api.rs

```diff
@@ -534,10 +534,11 @@ impl<AccountId: Eq + Hash> WalletSummary<AccountId> {
     /// Progress is represented in terms of the ratio between notes scanned and the total
     /// number of notes added to the chain in the relevant window. This ratio should only
     /// be used to compute progress percentages, and the numerator and denominator should
-    /// not be treated as authoritative note counts.
+    /// not be treated as authoritative note counts. The denominator of this ratio is
+    /// guaranteed to be nonzero.
```

Comment:
According to https://github.com/zcash/librustzcash/pull/1564#discussion_r1796339614 that is incorrect. That's okay for this release, but please file a PR to fix it.